WooCommerce/Classes/ViewRelated/Orders/Order Details/Issue Refunds/IssueRefundViewModel.swift

```diff
@@ -5,9 +5,30 @@ import Yosemite
 ///
 final class IssueRefundViewModel {
 
-    /// Order to be refunded
+    /// Struct to hold the necesary state to perform the refund and update the view model
     ///
-    private let order: Order
+    private struct State {
+        /// Order to be refunded
+        ///
+        let order: Order
+
+        /// Current currency settings
+        ///
+        let currencySettings: CurrencySettings
```

Comment:
I was wondering why `order` and `currencySettings` are part of the `State` while `shouldRefundShipping` is the only property that could change by the user? Would it make sense if these two properties belong to `IssueRefundViewModel` instead?

Reply:
I added them in the `state` struct because they are the immutable part of the state, and they are needed together with `shouldRefundShipping` to compute the properties that the view model uses to render the screen. 

I'm trying to have those two concerns separated. In the state struct, the underlying data. In the VM the data that is going to be presented to the user.